### DISCUSSION TITLE
Old Timer Dropping By

### DISCUSSION THREAD

**Discussion by Dank-Elf-Boy** [Monika-After-Story/MonikaModDev#8871] [appreciation]
It's been a while since I've left another "issue" here haha. So I'll just call this one a review of some of the changes I've seen since the last time I was active. 

I just want to go ahead and thank the head boss Mr. PotatoGuy for continuing this mod as well as the other coders, artists, and contributors! Hell, most of these mods have been abandoned ever since the decline in popularity that DDLC has had (or at least, since it's peak). You guys continue to put forth the effort to keep the adorable, sweet life of Monika alive in all our hearts. It's wholesome, it really is haha! 

I love the new artwork that has been done on Monika. I always felt that she could use a little extra 3D art to make her look more lively like the JY team did. And you guys nailed it. Speaking of the JY team, I think you guys have shown a lot more consistency towards your content than they have. Keep up the great work and keep getting that Python experience! 

I recently started my first year of college at the beginning of this year and am learning the ever so lovely C++ .... (it brings me depression lol) What I'm trying to say is, that this mod is what eventually sparked my interest in programming/computer science. I remember one of Monika's lines about "artificial intelligence" it's an area in computer science I will really look forward to. I guess I'll join the fight in created a sentient Monika one day, haha!

Anyways, thanks for reading~

Your boy,

**Bam8647**

**Comment by ThePotatoGuy** [comment]
[closing as appreciated]
